tests/unit/test_requests.py

```diff
@@ -88,8 +91,12 @@ def test_interface_redis(self, mock_get):
                                                 status_code=200)
         assert list(requests_cache_01)
         assert 'foo' in requests_cache_01
+
         assert requests_cache_01['foo'].content == 'bar'.encode()
         assert requests_cache_01['foo'].status_code == 200
+
+        assert requests_cache_01.__sizeof__() == RAND_CACHE_SIZE
```

Comment:
let's avoid acessing private methods where possible. `getsizeof()` should do it, right?

Reply:
no it doesnt. so getsizeof adds some overhead, about 24 bytes. initially the test was basically doing:
` assert sys.getsizeof(requests_cache_01)== RAND_CACHE_SIZE + 24`
and i dont know what exactly the extra 24 are. i can do something like get the size of the cache with size intialized to zero and that should give me the 24.
so:
`overhead = sys.getsizeof(RequestsCache(size=0))
assert sys.getsizeof(requests_cache_01)== RAND_CACHE_SIZE + overhead`

Reply:
i just really wanted to avoid writing the number 24 because i have a feeling when we modify the RequestsCache class that number will change...

Reply:
Right, it should be from the class itself. I'm OK with the current version, thanks for explaining.